api/capz/v1alpha3/azure_cluster_types.go

```diff
@@ -0,0 +1,32 @@
+package v1alpha3
```

Comment:
if we are going to use capi v1alpha4, I'd say let's go ahead and use capz v1alpha4 as well.

Reply:
I'm updating CAPI in the OpenStack PR so I will also update CAPZ to v1alpha4 in that one instead of here.